UltraStar Play/Assets/Scenes/PartyMode/PartyModeVersus/PartyModeVersusControl.cs

```diff
@@ -0,0 +1,169 @@
+using System.Collections;
+using PrimeInputActions;
+using ProTrans;
+using UniInject;
+using UniRx;
+using UnityEngine;
+using UnityEngine.UIElements;
+using Random = UnityEngine.Random;
+
+// Disable warning about fields that are never assigned, their values are injected.
+#pragma warning disable CS0649
+
+// Scene that shows before a round in Party Mode
```

Comment:
- I think this is a good idea. Players can prepare, change mics, etc. in this scene.
- However, maybe this could be combined with the round results?
    - After each round, the results are shown AND next players can prepare. I don't think we need a dedicated scene for this.